paper-badge.html

```diff
@@ -153,11 +153,21 @@
         icon: {
           type: String,
           value: ''
+        },
+
+        /**
+         * Treated as the hidden attribute.
+         */
+        hidden: {
```

Comment:
What, why? `hidden` is on every instance of `HTMLElement`, no? I don't think this is needed (but maybe a style for `[hidden]` is?